go.mod

```diff
@@ -87,7 +87,7 @@ require (
 	github.com/clbanning/mxj v1.8.4
 	github.com/cloudfoundry-community/go-cfclient v0.0.0-20210621174645-7773f7e22665
 	github.com/containerd/cgroups v1.0.4
-	github.com/containerd/containerd v1.6.8
+	github.com/containerd/containerd v1.6.9
```

Comment:
Could you bump this to 1.6.12 now that it's released? (to include a CVE fix as well)

Reply:
yes let me take a look

Reply:
cc @olivielpeau @vboulineau

Reply:
@paulcacheux sorry for only responding now, but the Agent uses Go 1.19 now on `main` (in https://github.com/DataDog/datadog-agent/pull/14930), could you pick this up again?